src/NzbDrone.Api.Test/Qualities/QualityDefinitionResourceValidatorTest.cs

```diff
@@ -0,0 +1,99 @@
+﻿using FluentValidation.TestHelper;
+using NUnit.Framework;
+using NzbDrone.Core.Qualities;
+using Sonarr.Api.V3.Qualities;
+
+namespace NzbDrone.Api.Test.Qualities;
```

Comment:
This should be moved under `v3`

Reply:
Rider is very intrusive about how it wants namespaces to be done. When I created the test fixture, it assumed the namespace should follow the directory structure. I'll take a look at some of the other tests and try to deduce what the namespace should be here.

I do wish there was a way to tell Rider to use a specific namespace for all files; I haven't quite found a good solution to that.

Reply:
Ah I misunderstood -- you meant move it under the directory named `v3`. Namespaces following the directory structure *does* seem to be the desired pattern for at least this test project. I will get it done!

Reply:
Ahh yeah, I was unclear, move to v3 folder + update namespace.